Userland/Demos/Mandelbrot/Mandelbrot.cpp

```diff
@@ -103,11 +103,11 @@ class MandelbrotSet {
             iteration++;
         }
 
-        if (iteration == max_iterations)
+        if (iteration == 1 || iteration == max_iterations)
```

Comment:
If you zoom out really far then the `1 + iteration - log(lz / log(2)) / log(2)` term becomes negative because of `Z(n)`s becoming too big. This causes a crash because `hue` will not be `>= 0 && < 360`.

I do not see this problem faced by others on the internet. I guess the reason is they do not support zooming out really far.

Two more ways to fix this:
- clamp the output to `0`.
- don't allow zooming out really far

Reply:
I chose to clamp the output (`mu`) to `0`.

Reply:
That makes more sense to me, the previous solution would have returned a black color for negative mu.